packages/auth/src/graphql/resolvers/grant.ts

```diff
@@ -17,7 +17,8 @@ export const getGrants: QueryResolvers<ApolloContext>['grants'] = async (
   ctx
 ): Promise<ResolversTypes['GrantsConnection']> => {
   const grantService = await ctx.container.use('grantService')
-  const grants = await grantService.getPage(args.input)
+  const { filter, input: pagination } = args
+  const grants = await grantService.getPage(pagination, filter)
   const pageInfo = await getPageInfo(
     (pagination: Pagination) => grantService.getPage(pagination),
```

Comment:
I think the `pageInfo` will be innacurate here because the actual page is filtering, but the `getPage` call that `pageInfo` uses does not filter. I think we want to pass in `(pagination: Pagination) => grantService.getPage(pagination, filter)` to `pageInfo` instead.

See https://github.com/interledger/rafiki/blob/main/packages/backend/src/graphql/resolvers/webhooks.ts. 

This is beyond the scope of this task, but just as an aside, I think this a good reason for returning `pageInfo` from `getPage` itself IMO. Instead of deriving it from a separate function call. I believe that was what was suggested here https://github.com/interledger/rafiki/pull/342#discussion_r879841635

Reply:
Good catch :+1: